dependencies.gradle

```diff
@@ -228,6 +229,8 @@ ext {
             firebaseCore:       "com.google.firebase:firebase-core:${firebaseVersion}",
             firebaseDb:         "com.google.firebase:firebase-database:${firebaseVersion}",
             firebaseMsging:     "com.google.firebase:firebase-messaging:${firebaseVersion}",
+
+            coroutine:          "org.jetbrains.kotlinx:kotlinx-coroutines-core:${kotlinCoroutine}",
```

Comment:
In this PR, have you already used this dependency?

Reply:
No, I've tried using it, but in the current case is inconvenient. I think near future I need it.